test/test-lib/fixtures.ts

```diff
@@ -113,6 +148,64 @@ const loaders: Dictionary<LoaderFunc> = {
 	},
 };
 
+loaders['devices'] = async (jsonData, fixtures) => {
+	const user = await fixtures.users[jsonData.belongs_to__user];
+	if (user == null) {
+		logErrorAndThrow(`Could not find user: ${jsonData.user}`);
+	}
+	const application = await fixtures.applications[
+		jsonData.belongs_to__application
+	];
+	if (application == null) {
+		logErrorAndThrow(
+			`Could not find application: ${jsonData.belongs_to__application}`,
+		);
+	}
+
+	const requestOpts = buildApiRequestOpts({
```

Comment:
Can you use `createResource` like the other loaders please

Reply:
yep updated